### PR TITLE
Fixes #10 add Temperature dimension

### LINKED ISSUE #10: Add Temperature as Dimension
Right now there is only Kelvin and I think this is not how it should be
What about we have a Temperature dimension with Kelvin, Celcius, Fahrenheit etc..

@Yuri05 @UlrichSi. thoughts?

Temperature is a required input for the upcoming dermal model. We might as well get the dimension in asap

### PR DESCRIPTION
**Pull request by msevestre**

### CODE REVIEW

File: OSPSuite.Dimensions.xml [hunk 1]
```diff
@@ -739,6 +739,14 @@
         <Unit name="µmol/min" factor="1" />
       </Units>
     </Dimension>
+    <Dimension name="Temperature" baseUnit="K" defaultUnit="K">
```
Comment (msevestre) [inline OSPSuite.Dimensions.xml]: I kept Kelvin for compatibility reasons. But this Dimension does not make sense

File: OSPSuite.Dimensions.xml [hunk 2]
```diff
@@ -739,6 +739,14 @@
         <Unit name="µmol/min" factor="1" />
       </Units>
     </Dimension>
+    <Dimension name="Temperature" baseUnit="K" defaultUnit="C">
```
Comment (msevestre) [inline OSPSuite.Dimensions.xml]: @UlrichSi @Yuri05 
Do you agree on C has being the default unit? Or should it be K? (internal unit will always be K)

**Comment by UlrichSi** [comment]
I think the implementation is ok in °C. - But not just "C", since this  could be mistaken with the electrical unit "Coulomb", officially abbreviated "C".

**Comment by UlrichSi** [comment]
If "°C" causes problems, use "deg C" instead.

**Comment by msevestre** [comment]
°C should work Good catch btw. Having two dimension with the same unit is not good

**Comment by msevestre** [comment]
@UlrichSi Shouldn't we use °F as well? seems to be usual in most research on google I did

**Comment by UlrichSi** [comment]
@msevestre Absolutely "°F"! "F" would be "Farad", the unit of electrical capacitance.

File: OSPSuite.Dimensions.xml [hunk 3]
```diff
@@ -739,6 +739,14 @@
         <Unit name="µmol/min" factor="1" />
       </Units>
     </Dimension>
+    <Dimension name="Temperature" baseUnit="K" defaultUnit="°C">
+      <BaseRepresentation lengthExponent="0" massExponent="0" timeExponent="0" electricCurrentExponent="0" temperatureExponent="1" amountExponent="0" luminousIntensityExponent="0" />
+      <Units>
+        <Unit name="°C" factor="1" offset="273.15" />
+        <Unit name="°F" factor="5/9" offset="255.37" />
```
Comment (Yuri05) [inline OSPSuite.Dimensions.xml]: Should we define the offset as `459.67*5/9` instead of `255.37`?

Comment (msevestre) [reply OSPSuite.Dimensions.xml]: why?

Comment (Yuri05) [reply OSPSuite.Dimensions.xml]: BEcause that's the exact definition.
@UlrichSi Thoughts?

Comment (msevestre) [reply OSPSuite.Dimensions.xml]: don't you thik that 459.67 is an approximation lol

Plus if you look at def, you can also find
(32°F − 32) × 5/9 + 273.15

**Comment by UlrichSi** [comment]
Maybe we better should display a warning if somebody selects "°F" as unit. "Enter at your own risk", or something like that. 

Seriously, I am not sure if a precision better than 1/10 degree is needed.

**Pull request merged**